exonum/src/proto/schema/exonum/proofs.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum;
+
+option java_package = "com.exonum.core.messages";
+
+import "blockchain.proto";
+import "consensus.proto";
+import "map_proof.proto";
+
+message BlockProof {
```

Comment:
Please add documentation

Reply:
I'd rather copy-paste the docs of the Rust structs here, because these docstrings will be used for auto-generated documentation of the protobuf classes in Java